os/binfmt/binfmt_loadbinary.c

```diff
@@ -90,7 +90,16 @@ int load_binary(int binary_idx, FAR const char *filename, load_attr_t *load_attr
 #ifdef CONFIG_APP_BINARY_SEPARATION
 	/* Allocate the RAM partition to load the app into */
 	uint32_t *start_addr;
+#ifdef CONFIG_ARMV7M_MPU
```

Comment:
@jeongchanKim
I have added a small hack to the estimation of size for ram partition.
As you know, since we use memalign API the ram size required is much larger than what is calculated by the mkbinaryheader script. For the time being I have multiplied the size by 3 times and it seems to be working fine. But we need to find a better way to estimate the RAM size. Please let me know if you have any suggestions.